pcsx2/GS/Renderers/Common/GSDevice.h

```diff
@@ -43,6 +47,14 @@ enum class ShaderConvert
 	Count
 };
 
+enum class SetDATM
```

Comment:
```suggestion
enum class SetDATM : u8
```
(that way it can be packed in the bitfield with type safety)